### ISSUE TITLE
Better document after hook()

### ISSUE THREAD

**Issue by lewisf** [webdriverio/webdriverio#1844]
## The problem

Better document the arguments of hooks (particularly the after hook) to facilitate the writing on services.

## Environment

* WebdriverIO version: 4.6.2
* Node.js version: 6.7
* [Standalone mode or wdio testrunner](http://webdriver.io/guide/getstarted/modes.html):
* if wdio testrunner, running synchronous or asynchronous tests: synchronous
* Additional wdio packages used (if applicable): n/a

## Details

I'm trying to write a service and use the after hook. The developer guide test runner configuration section just has this bit:

```
    // Gets executed after all tests are done. You still have access to all global variables from
    // the test.
    after: function (result, capabilities, specs) {
    },
```

After playing around with it a bit, it looks like a result of 0 means the tests passed while a result of 1 means the tests failed. Is that right? If so, I'd love to help update the docs somewhere to reflect that.

## Link to Selenium/WebdriverIO logs

n/a

## Code To Reproduce Issue [ Good To Have ]

n/a

**Comment by christian-bromann** [comment]
> I'd love to help update the docs somewhere to reflect that.

Yes, feel free to send a PR

**Comment by ififthelement** [comment]
Hi,

I have also checked the Config file available at  http://webdriver.io/guide/testrunner/configurationfile.html.

I tried the afterScenario and after hook. But I have noticed that after scenario execuction or the enter execution, the simplest console.log in these hooks never got exceuted. Am I missing something here !?

`	afterScenario: function(scenario) {
		console.log("after scenario " + scenario.getName());
	}`
